src/app/shared/forms/planet-tag-input-dialog.component.ts

```diff
@@ -98,7 +99,7 @@ export class PlanetTagInputDialogComponent {
     this.selected.clear();
     this.data.reset(this._selectMany);
   }
-
+  
```

Comment:
We shouldn't have a line with only spaces.  If you're using VS Code you can set the editor to remove these extra spaces automatically.